diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -373,6 +373,15 @@ namespace diff_drive_controller{
   {
     if (isRunning())
     {
+      // check that we don't have multiple publishers on the command topic
+      if (sub_command_.getNumPublishers() > 1)
```

Comment:
@Karsten1987 Could you add the `allow_multiple_cmd_vel_publishers` param?

`if (!allow_multiple_cmd_vel_publishers_ && sub_command_.getNumPublishers() > 1)`